cgi-bin/LJ/Console/Command/ScreenList.pm

```diff
@@ -0,0 +1,67 @@
+# This code was forked from the LiveJournal project owned and operated
+# by Live Journal, Inc. The code has been modified and expanded by
+# Dreamwidth Studios, LLC. These files were originally licensed under
+# the terms of the license supplied by Live Journal, Inc, which can
+# currently be found at:
+#
+# http://code.livejournal.org/trac/livejournal/browser/trunk/LICENSE-LiveJournal.txt
+#
+# In accordance with the original license, this code and all its
+# modifications are provided under the GNU General Public License.
+# A copy of that license can be found in the LICENSE file included as
+# part of this distribution.
```

Comment:
Given how very little I am changing in these commands, it is unclear to me whether or not the original LJ wording should apply. I can use the updated DW wording, but it seems...disingenuous...to imply myself as Sole Author.